sys/mips/conf/std.CHERI_PURECAP

```diff
@@ -0,0 +1,50 @@
+#
+# CHERI -- Kernel for the SRI/Cambridge "CHERI" (Capability Hardware Enhanced
+# RISC Instructions) FPGA soft core, as configured in its Terasic DE-4
+# reference configuration - 128bit version.
+#
+# $FreeBSD$
+#
+
+include		"std.CHERI"
+
+machine		mips mips64c128
+
+nomakeoptions 	ARCH_FLAGS
+makeoptions 	ARCH_PIC_FLAGS="-fpic"
+makeoptions 	ARCH_FLAGS="-mcpu=cheri128 -mxcaptable"
+makeoptions 	CFLAGS+="-gdwarf-5"
+makeoptions 	WERROR="-Wno-error"
+makeoptions 	KERNLOADADDR=0xffffffff80100000
+makeoptions 	CHERI_USE_CAP_TABLE=plt
+makeoptions	CHERI_SUBOBJECT_BOUNDS=subobject-safe
+makeoptions   LDSCRIPT_NAME=ldscript.mips.cheri
```

Comment:
Ditto

Reply:
I'm tempted to keep these to have a more explicit configuration for those parameters, albeit a bit more verbose. I can move the linker script though.